site-content/source/modules/ROOT/pages/events.adoc

```diff
@@ -0,0 +1,277 @@
+= Events
+:page-layout: events
+:page-role: events
+
+////
+NOTES FOR CONTENT CREATORS
+- To add a new event card, copy and paste markup for one card below.  Copy from '//start' to the next '//end'
+- Replace event card fields: title, date, image, short text and external link.
+////
+
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Cassandra Summit 2023
+[discrete]
+==== March 13-14, 2023
+------
+[openblock,card-content]
+------
+image:events/c_summit_logo.png[Cassandra Summit logo]
+
+Cassandra Summit is back! Let's meet in San Jose, California, to celebrate what we built together and share what's coming next.
+
+[openblock,card-btn card-btn--events]
+--------
+[.btn.btn--alt]
+https://events.linuxfoundation.org/cassandra-summit/[Go to event,window=_blank]
```

Comment:
When we link to external non-ASF hosted sites, is there any need for a notification that you're leaving the ASF space and going to a particular vendor or other foundation's properties / sites? Genuinely asking here as I don't know and want to make sure we grease the skids on this as much as possible.

Reply:
Hmm, I don't know the answer. I honestly assumed that external links would be good out-of-the-box here as they are on blog posts.
(a general warning "you're leaving the site" would be best handled by client-side code, which would be a feature to implement independently from individual external links).
I took care to have external links open in new tab all the time.

Reply:
👍 Makes complete sense to me. What triggered the thought was actually a completely different domain that's regulated by a ratings board etc, so quite different context. But figured I'd check.